lib/src/workspace.rs

```diff
@@ -15,6 +15,8 @@
 #![allow(missing_docs)]
 
 use std::collections::HashMap;
+#[cfg(windows)]
+use std::ffi::CString;
```

Comment:
nit: I think we usually move conditional imports inside the conditional block where they're needed. So move these imports to line 132 and drop the attribute on it.